geode-core/src/main/java/org/apache/geode/distributed/internal/AbstractDistributionConfig.java

```diff
@@ -1360,6 +1360,8 @@ public static Class _getAttributeType(String attName) {
     m.put(SECURITY_POST_PROCESSOR,
         "User defined fully qualified class name implementing PostProcessor interface for integrated security. Defaults to \"{0}\". Legal values can be any \"class name\" implementing PostProcessor that is present in the classpath.");
 
+    m.put(SSL_USE_DEFAULT_PROVIDER, "If true allows clients to provide a custom java.security.Provider. When using this clients do not need to configure other ssl properties except ssl-enabled-components and ssl-require-authentication.");
```

Comment:
This isn't specific to client caches is it?  I think you should reword this.  Maybe
"If true allows you to provide a custom java.security.Provider.  When using this you do not need to configure other ssl properties except ssl-enabled-components and ssl-require-authentication."

Reply:
Yes, this solution can work for the server if users want to use a custom provider. I will rephrase this with docs help.